app/src/pages/inside/common/suiteTestToolbar/actionPanel/actionPanel.scss

```diff
@@ -22,3 +22,32 @@
     display: none;
   }
 }
+
+@media (min-width: 768px) and (max-width: 1195px) {
```

Comment:
Please, use appropriate screen size variables.
See the `common/constants/css/variables/screen-size-variables.scss`

Reply:
Some of the constant values don't match what I need to use, but I will use what I can find in there.